playbook-servers-docker.yml

```diff
@@ -0,0 +1,30 @@
+---
+- hosts: docker_servers
+  remote_user: sirius
+  become: true
+
+  pre_tasks:
+    - name: Include distribution-dependent variables
+      include_vars: "{{ item }}"
+      vars:
+        possible_var_files:
+          - "group_vars/{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml"
+          - "group_vars/{{ ansible_distribution }}.yml"
+          - "group_vars/{{ ansible_os_family }}.yml"
+      loop: "{{ q('first_found', possible_var_files, errors='ignore') }}"
+
+  roles:
+    - role: geerlingguy.docker
+
+  tasks:
+    - name: Create docker certs.d/ for the internal registry
+      file:
+        path: "/etc/docker/certs.d/{{ docker_registry_server }}/"
+        state: directory
+        mode: "0755"
+
+    - name: Copy self signed crt to the client machine
+      ansible.posix.synchronize:
+        src: "{{ docker_registry_ssl_crt_path }}"
+        dest: "/etc/docker/certs.d/{{ docker_registry_server }}/ca.crt"
+      delegate_to: "{{ docker_registry_server }}"
```

Comment:
- Do we need to also put the certificate at `usr/local/share/ca-certificates/{{ docker_registry_server }}.crt`?
- I think we might need to issue a `update-ca-certificates` command. Or is this step not needed?
- I think we need to restart docker daemon service after changing its certificate

Reply:
Just copying the content to the corresponding folder is enough. The docker service will automatically detect the certificate.